src/scorecard.rs

```diff
@@ -99,13 +88,7 @@ impl fmt::Display for ScoreCardData {
 
 impl ScoreCardData {
     pub fn get_line_by_id(&self, zid: &LineId) -> &LineData {
-        let line = self.line.iter().find(|l| l.id == *zid);
-
-        // HELP: How can I do this without the match?
-        match line {
-            None => panic!("not found"),
-            Some(x) => x,
-        }
+        self.line.iter().find(|l| l.id == *zid).expect("not found")
```

Comment:
This is completely identical to what it replaced, so that's how you can do it without a match. However, you may just want to return the `Option<&LineData>` and have the code calling it deal with it instead.